Fix #50 - Broken docs

mmBabol opened the pull request:
Fixed broken doc issue from PR #35, with the CSS and images not showing up. 

In the `siteConfig.js` file:
 * fixed the baseUrl to `https://humphd.github.io/next/`
 * corrected favicon path to point to the `img/favicon/` directory

Added missing files/directories into the `.gitignore` file for Docusaurus

Removed obsolete values from the `en.json` file. The doc files have now been renames and we do not need the old values in here anymore.

Added missing `database.svg` image.

New doc page should load correctly with the proper css & images as shown below

![working home page](https://user-images.githubusercontent.com/13471358/40521123-e73d1ec2-5f97-11e8-8887-d2968cf55d77.PNG)


** Update **

* Removed ```src/editor/.DS_Store``` from ```.gitignore```
* To fix the auto-generated values in the ```en.json``` file, I had to fix the case inconsistency in all of the doc files. The generate ```en.json``` file is now generated correctly 
* Minified the SVG image file size

humphd commented on .gitignore:
```diff
@@ -60,6 +60,20 @@ typings/
 # Docusaurus build folder
 website/build/
 
+.DS_Store
```
You can probably get rid of:

```
#OSX .DS_Store files
 src/editor/.DS_Store
```

humphd commented on website/i18n/en.json:
```diff
@@ -9,18 +9,12 @@
     "community": "Community",
     "Community": "Community",
     "introduction": "Introduction",
-    "Next introduction": "Next introduction",
```
Let's do these changes in the src file vs. this one, which is generated.

humphd commented on website/i18n/en.json:
```diff
@@ -9,18 +9,12 @@
-    "doc4": "Other Document",
-    "doc5": "Fifth Document",
     "Introduction": "Introduction",
     "technologies": "Technologies",
```
Let's not double include lowercase and Uppercase versions of strings.

humphd commented on .gitignore:
```diff
@@ -60,6 +60,21 @@ typings/
+.DS_Store
+
+lib/core/metadata.js
+lib/core/MetadataBlog.js
```
These can go

humphd commented on .gitignore:
```diff
@@ -60,6 +60,21 @@ typings/
+website/yarn.lock
+website/node_modules
+website/i18n/*
+!website/i18n/en.json
```
Let's remove this, and also do `git rm -f website/i18n/en.json`

humphd commented on website/siteConfig.js:
```diff
@@ -14,7 +14,7 @@ const users = [
     caption: 'User1',
```
Let's get rid of the whole `users` array and see if it works, since we don't have other projects involved with ours yet!

mmBabol commented:
I fixed the issues that we discussed today:
* Removed duplicate ```.DS_Store``` variable from ```.gitignore``` and reorganized the docusaurus gitignore section
* Added ```en.json``` to ```.gitignore```
* Removed the ```en.json``` file
* Fixed capitalization inconsistencies in ```siteConfig.js``` for ```doc: 'Introduction'``` & ```doc: 'Community'```
